packages/nextra/src/icon-components/arrow-right.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ComponentProps } from 'react'
```

Comment:
I think it’s not good place for shared icons, since it’s not related to nextra, but for themes only

Reply:
My point is that 
 if user want to build their own themes. They could import icons from nextra.